src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/DataFlowPass.vb

```diff
@@ -615,13 +615,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             ' If the state of the slot is 'assigned' we need to clear it 
             '    and possibly propagate it to all the parents
+            If Me._tryState IsNot Nothing Then
```

Comment:
Nested try has a nested try state that is merged with the outer try state. Nothing had to be fixed there.
HoistUsing004 is a nested Using test.